db/migrate/20230829115907_update_school_workforce_censuses.rb

```diff
@@ -0,0 +1,42 @@
+class UpdateSchoolWorkforceCensuses < ActiveRecord::Migration[7.0]
+  def change
+    add_columns_if_not_exist(
+      :school_workforce_censuses,
+      {
+        school_urn: :integer,
+        contract_agreement_type: :string,
+        totfte: :integer,
+        subject_description_sfr: :string,
+        general_subject_code: :string,
+        hours_taught: :integer
+      }
+    )
+
+    (1..15).each do |n|
+      add_column_if_not_exist(
+        :school_workforce_censuses,
+        "subject_#{n}".to_sym,
+        :string
+      )
+    end
```

Comment:
Shouldn't these columns be removed? I can see they are being removed in the schema but added here instead? 🤔

Reply:
When I run `rake db:schema:load` all looks good, but `rake db:migrate` doesn't produce the same effect.